exercises/concept/lucians-luscious-lasagna/.meta/config.json

```
@@ -1,6 +1,9 @@
 {
-  "blurb": "TODO: add blurb for lucians-luscious-lasagna exercise",
+  "blurb": "Learn about methods by helping cook lasagna.",
```

It would be nice to keep this consistent with the [corresponding F# exercise](https://github.com/exercism/fsharp/blob/main/exercises/concept/lucians-luscious-lasagna/.meta/config.json): 

```suggestion
  "blurb": "Learn about the basics of C# by following a lasagna recipe.",
```